spitfire/runtime/__init__.py

```diff
@@ -68,15 +72,15 @@ def import_module_symbol(name):
     module = __import__(module_name, globals(), locals(), [symbol_name])
     try:
         symbol = getattr(module, symbol_name)
-    except AttributeError, e:
+    except AttributeError as e:
         raise ImportError("can't import %s" % name)
     return symbol
 
 
 # map template function names to python function names
 # inject them into a module so they run as globals
 def register_functions(module, template_function_map):
-    for t_name, f_name in template_function_map.iteritems():
+    for t_name, f_name in six.iteritems(template_function_map):
```

Comment:
We can use the python2/python3 compatible `iter(<map_value>)` approach here (combined with next suggestion).

```suggestion
    for t_name in iter(template_function_map):
```